src/components/resend-mfa-code/index.njk

```diff
@@ -22,7 +22,14 @@
         }) }}
 
         {% if support2hrLockout %}
-        <p class="govuk-body">{{'pages.resendMfaCode.paragraph1' | translate}}</p>
+        <p class="govuk-body">
+            {{'pages.resendMfaCode.paragraph1' | translate}}
+            {% if supportReauthentication %}
```

Comment:
This will always show the `reauthAction` text when reauth is enabled in the env variable.

Instead I think this needs to be aware of what user journey the user is on, and only show the `reauthAction` text if the user is on a reauth journey.

I'm having a look to see how you could determine the journey.

Reply:
I think you can use `Boolean(req.session.user.reauthenticate)` to know if they are on a reauth journey.

https://github.com/govuk-one-login/authentication-frontend/blob/7341b706c70dce9f0a8268a2251ce910c87773c9/src/components/authorize/authorize-controller.ts#L171

Reply:
Once changed, could you provide screenshots evidencing that reauth and non-reauth journeys show the differing texts? Thanks @di-fabs!

Reply:
I pushed the changes.

![Screenshot 2024-07-24 152823](https://github.com/user-attachments/assets/5032fdf4-c038-4784-9e16-d0ecc20bf0de)
![Screenshot 2024-07-24 152805](https://github.com/user-attachments/assets/e41ea218-92e6-4893-9aef-e1d8d12112c5)